components/eam/bld/namelist_files/use_cases/2000_cam5_av1c-04p2-MMF-1mom.xml

```diff
@@ -88,7 +88,7 @@
 
 <!-- Prescribed aerosols -->
 <prescribed_aero_datapath>atm/cam/chem/trop_mam/aero</prescribed_aero_datapath>
-<prescribed_aero_file>mam4_0.9x1.2_L72_2000clim_c170323.nc</prescribed_aero_file>
```

Comment:
If these new files were made using the F2010 compset, do we really want to update compsets based on F2000?

Reply:
Fair point about merge conflicts in the future. I think @PeterCaldwell's reasoning behind changing these files as well was that we don't know what simulation produced the original prescribed aerosol file, so it's better to use one that's not quite consistent (2000 vs 2010). 
For the scream F2000 compset, I do think that we should use the newer aerosol field because we know what kind of model/simulation they came from. For the non-scream compsets, I don't have a strong opinion so I can revert them as well. Any thoughts, @PeterCaldwell?

Reply:
The merge conflict question is a good one... but irrelevant for the dyamond2 branch we're working on now because we will never merge upstream into it. The 2000 vs 2010 issue is legit, but I'm pretty sure that old prescribed_aero_file was created by @bogensch  from some random v1 alpha run with no real provenance... which is why I thought it would be better to switch to a maint1.0 tag even if we use 2000 instead of 2010. The other issue is that only a Luddite would use an F2000 compset these days so they deserve to get an inconsistent prescribed aerosol file. All in all, I don't care what we do for these F2000 cases (because I'm not a Luddite) and think whatever allows us to merge this PR and move on to other things is best. Sorry for suggesting you make a change that doesn't really matter, Chris!

Reply:
Ok, reverting back to old file: if people want to use an old F compset, they should expect to get an old/ratty aerosol file.